libs/antsi/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "antsi"
+version = "0.0.0-reserved"
+edition = "2021"
```

Comment:
There are a bunch of keys missing here. Could you please add the same keys as we have in `error-stack`? A few notes:
- MSRV should probably be `1.63` or older if possible, so we could use it in `error-stack`. Older is better, but not needed IMO
- `keywords` should contain `"no_std"` and `categories` should contain `"no-std"`
- `publish` has to be set to be set to `true` of should be removed

Reply:
> MSRV should probably be `1.63` or older if possible [..]

The reserved version will be `1.63`, once done I will use [`cargo-msrv`](https://github.com/foresterre/cargo-msrv) to determine the lowest MSRV we can go (and see if we could go lower)

> `keywords` should contain `"no_std"` and categories should contain `"no-std"`

Sure no problem. @nonparibus any thoughts on other categories and keywords it should contain?

Reply:
fixed in [418e322](https://github.com/hashintel/hash/pull/2018/commits/418e3228a5da5f4b907b27f7887646e9b571ad53)